src/sentry/static/sentry/app/views/dashboardsV2/widgetCard.tsx

```diff
@@ -487,3 +488,22 @@ class WidgetCardVisuals extends React.Component<WidgetCardVisualsProps> {
     );
   }
 }
+
+const StyledTransparentLoadingMask = styled(props => (
+  <TransparentLoadingMask {...props} maskBackgroundColor="transparent" />
+))`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const LoadingScreen = ({loading}: {loading: boolean}) => {
```

Comment:
Is this style of loader something we should be using with other charts as well?

Reply:
I had borrowed this from Discover query cards.

It seems the pattern for pages where it has only 1 chart, the transparent mask is used instead; while letting the chart remain interactive.

Reply:
Hm... would this result in way too many spinners in different positions for other pages?